README.md

```diff
@@ -1,4 +1,4 @@
-# grunt-contrib-watch v0.6.1 [![Build Status: Linux](https://travis-ci.org/gruntjs/grunt-contrib-watch.png?branch=master)](https://travis-ci.org/gruntjs/grunt-contrib-watch)
+# grunt-contrib-watch v0.7.0 [![Build Status: Linux](https://travis-ci.org/gruntjs/grunt-contrib-watch.png?branch=master)](https://travis-ci.org/gruntjs/grunt-contrib-watch)
```

Comment:
I think this was updated for the purpose of our fork. The first commit is probably the only one needed and versioning can be handled whenever you guys do the next release.